doc/api/index.rst

```diff
@@ -0,0 +1,20 @@
+API Documentation
+=================
+
+.. toctree::
+
+   functions
+   streams
+   raw_streams
+   misc
+   settings
+
+.. automodule:: sounddevice
+   :undoc-members:
+   :exclude-members: RawInputStream, RawOutputStream, RawStream,
+                     InputStream, OutputStream, Stream,
+                     CallbackFlags, CallbackStop, CallbackAbort,
+                     PortAudioError, DeviceList,
+                     AsioSettings, CoreAudioSettings, WasapiSettings
```

Comment:
None of those options have an effect, right?

I think they can all be removed.

Reply:
Done.

Reply:
You left one option in there?

Reply:
Ah, right. Removed.